docs/troubleshooting.asciidoc

```diff
@@ -28,3 +28,72 @@ include::copied-from-beats/getting-help.asciidoc[]
 == Debug
 
 include::copied-from-beats/debugging.asciidoc[]
+
+[[common-problems]]
+== Common problems
+[[queue-is-full]]
+=== 503: Queue is full
+APM Server has an internal queue which stores documents until they can be delivered to Elasticsearch.
+It is designed this way to alleviate problems that might occur if Elasticsearch is intermittently unavailable or if a large spike of data arrives at the APM Server at the same time.
```

Comment:
I assume with `the internal queue` you are summarizing all queues in place until data are written to ES. If so I suggest to also add that the queue is used to send data in batches to ES for optimized write.